scripts/test-install.sh

```diff
@@ -1,41 +1,109 @@
 #!/usr/bin/env sh
 set -e
 
+# All tests are headed by a echo 'Test'.
+# The general schema is:
+# 1. Check we can install from the tarball.
+# 2. The install script works and correctly exits without errors
+# 3. Requiring/importing Puppeteer from Node works.
+
+## Puppeter tests
+
+echo "Setting up Puppeteer tests..."
 ROOTDIR="$(pwd)"
-# Pack the module into a tarball
 npm pack
 tarball="$(realpath puppeteer-*.tgz)"
+
+echo "Testing... Chrome CommonJS"
 TMPDIR="$(mktemp -d)"
 cd $TMPDIR
-# Check we can install from the tarball.
-# This emulates installing from npm and ensures that:
-# 1. we publish the right files in the `files` list from package.json
-# 2. The install script works and correctly exits without errors
-# 3. Requiring Puppeteer from Node works.
 npm install --loglevel silent "${tarball}"
 node --eval="require('puppeteer')"
+node --eval="require('puppeteer/lib/cjs/puppeteer/revisions.js');"
 ls $TMPDIR/node_modules/puppeteer/.local-chromium/
 
-# Again for Firefox
+echo "Testing... Chrome ES Modules"
+TMPDIR="$(mktemp -d)"
+cd $TMPDIR
+echo '{"type":"module"}' >>$TMPDIR/package.json
+npm install --loglevel silent "${tarball}"
+node --input-type="module" --eval="import puppeteer from 'puppeteer'"
+node --input-type="module" --eval="import 'puppeteer/lib/esm/puppeteer/revisions.js';"
+node --input-type="module" --eval="
+import puppeteer from 'puppeteer';
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto('http://example.com');
```

Comment:
For testing whether `puppeteer` works in cjs and esm, do we need to run a real test or would having import and maybe a few simple steps without network request enough? (like only `const await browser.newPage(); await browser.close();`)

Reply:
Not too sure, but it doesn't hurt to run a real test as this provides more coverage inherently.

Reply:
hihiim, I think we don't have a clear win there with running a real test for testing the install and whether the exports we provide works. So having the additional overhead of making a network request in this test and depending on an external website is not justified.

Thus, I'd prefer removing those parts that goes to a website and takes a screenshot. WDYT?

Reply:
I think it's okay to use example.com for this smoke test as screenshooting touches deeper dependencies. So I think it's fine to keep the test as is. If it becomes flaky, we can reduce the test area.